ci/scenarios/vPool/add_remove_vpool_concurrently/main.py

```diff
@@ -35,32 +35,33 @@ class VPoolTester(CIConstants):
     LOGGER = Logger('scenario-{0}'.format(TEST_NAME))
 
     def __init__(self):
-        self.LOGGER.info("Initializing concurrent vpool testing.")
-        self.valid_storagerouters = []
+        VPoolTester.LOGGER.info("Initializing concurrent vpool testing.")
+        VPoolTester.valid_storagerouters = []
```

Comment:
Won't reset this instance its valid storagerouters but rather the static instances